src/Compilers/CSharp/Portable/Symbols/Source/SourceClonedParameterSymbol.cs

```diff
@@ -56,6 +56,8 @@ internal override bool IsMetadataOptional
             }
         }
 
+        internal override BoundExpression ExplicitDefaultExpression => _originalParam.ExplicitDefaultExpression;
```

Comment:
>Can you explain why null is the more appropriate result, for both this case and WrappedParameterSymbol? I'm not sure I understand the reasoning.

Because this parameter is cloned from some real parameter and itself doesn't have any corresponding syntax. Whatever analysis should be performed on the value will be performed on the original declaration and I don't believe there is a reason to duplicate that analysis. If you believe otherwise, you should try to demonstrate that with specific scenario.